tailor_api/Cargo.toml

```
@@ -10,3 +10,10 @@ repository = "https://github.com/AaronErhardt/tuxedo-rs"
 [dependencies]
 atoi = "2"
 serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+
+[dev-dependencies]
+tailor_api = { version = "0.2.1", path = ".", features = ["test-utilities"] }
+
+[features]
+test-utilities = []
```

We don't need serde-json if the "test-utilities" feature isn't enabled.

```suggestion
serde_json = { version = "1", optional = true }

[dev-dependencies]
tailor_api = { version = "0.2.1", path = ".", features = ["test-utilities"] }

[features]
test-utilities = ["serde_json"]
```

---

Thanks! TIL...